### ISSUE TITLE
Twin X14

### ISSUE THREAD

**Issue by 10KPP** [FrSkyRC/ETHOS-Feedback-Community#4379]
Hello, is it possible to set the trim for the throttle to Pot2?

**Comment by mawzthefinn** [comment]
Yes, but it's a bit of a weird workaround. There's no way to directly set the trim, but if you do the following, you will get trim on Pot2 with the same behaviour as low position trim in the throttle. This requires ETHOS 1.5 or later

1. Create a Free Mix, name it Throttle Trim, source is Pot2, Output is CH64
2. Add a curve action to the Free Mix, edit the curve, name it Throttle Trim, set it to 2 points, set the points to -100 and -50
3. Add a curve to your throttle mix, name it Throttle, set to 2 points, set point 2 to 100, long-press point 1, select 'use a source', select Channels->Ch64

You now have low position trim on the throttle,.

**Comment by 10KPP** [comment]
Perfect, thank you.

**Comment by BladeScraper-Designs** [comment]
https://github.com/FrSkyRC/ETHOS-Feedback-Community/issues/4101

**Comment by mawzthefinn** [comment]
Closing this issue as the long-term fix is #4101
